nix/hcloud/default.nix

```diff
@@ -90,7 +90,7 @@ in
 
         provisioner.remote-exec = {
           inline = [
-            "curl https://raw.githubusercontent.com/elitak/nixos-infect/master/nixos-infect | PROVIDER=hetznercloud NIX_CHANNEL=nixos-23.11 NO_REBOOT=true bash 2>&1 | tee /tmp/infect.log"
```

Comment:
i wonder if we should hold back with `hetzner` until [this](https://github.com/elitak/nixos-infect/issues/207) issue is triaged or resolved? :thinking: